docs/azure-infrastructure.md

```diff
@@ -0,0 +1,47 @@
+# Apply for Teacher Training - Azure Infrastructure 
+
+## Purpose
+
+This document describes the Azure architecture that the Apply for Teacher Training service is deployed into.
+
+## Azure subscriptions
+
+The apply service is deployed across three subscriptions located in the "DfE Platform Identity" tenant, otherwise known as the CIP tenant.
+
+![Azure Infrstructure - Subscriptions](architecture-subscriptions.png)
+
+Within each subscription there are a number of resource groups. In Azure a resource group is simply a logical grouping of resources for ease of deployment, management and monitoring etc. The entire Apply service is contained in a single resource group with multiple instances of the application deployed across multiple resource groups as shown in the diagram above. All of these deployed instances are identical but wholly independent of each other. In addition to the two application instances there are some further resource groups that are required for other administrative reasons. These are:
```

Comment:
The sentence 

> The entire Apply service is contained in a single resource group with multiple instances of the application deployed across multiple resource groups as shown in the diagram above

confused me a bit (not hard to do) - at first reading it seemed like there was a single 'container' resource group for the 'entire service' that contains other resource groups but I'm not sure. I guess you mean that an entire Apply service instance is contained in a single resource group?